README.md

```diff
@@ -1,68 +1,31 @@
-# kruize-ui
+# Kruize-UI
 
-This repository contains the code for Kruize Autotune user interface. The objective behind creating the UI is to enable newbies to explore and contribute to Autotune and simplify the initial steps of starting an Autotune experiment. The UI significantly reduces the burden on user to understand, operate and work with Autotune. Not only this it also helps better explain and present what remarkable optimizations Autotune can bring to site reliability engineering.
+This repository contains the code for Kruize user interface. The objective behind creating the UI is to enable users to explore and contribute to Kruize. The UI significantly reduces the burden on user to understand, operate and work with Kruize. Not only this it also helps better represent the remarkable optimizations Kruize can bring to your optimization usecases.
 
-## Quick Start
-
-### Clone Repository
+This application is built using [Patternfly and React](https://www.patternfly.org/v4/get-started/developers)
 
-```
-git clone git@github.com:kruize/kruize-ui.git
-```
+## Before You Start
 
-### Go to the kruize folder
+Kruize UI needs Kruize server running locally in the remote monitoring mode. You can setup Kruize [from here](https://github.com/kruize/kruize-demos/tree/main/remote_monitoring_demo). 
 
-```
-cd kruize-ui
-```
+Additionally Kruize UI can directly be accessed once remote monitoring demo is setup.
 
-### Switch to mvp_demo branch
+In case you want to run UI locally you would need the Cluster IP and Kruize Port number. Make use of the following commands:
+1. Cluster IP : In case of a minikube cluster ```minikube ip```
+2. Kruize Port Number : ```kubectl -n monitoring  get svc kruize --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort 2>/dev/null```
```

Comment:
Please convert this to commands that can be copy pasted
```
export CLUSTER_IP=$(minikube ip)
 export KRUIZE_PORT=$(kubectl -n monitoring  get svc kruize --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort 2>/dev/null)
```

Reply:
done!